packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -56,7 +86,7 @@ window.\$hotReloadHook = function(modules) {
         if (reloadCount == modules.length) {
           require(["$entrypoint", "dart_sdk"], function(app, dart_sdk) {
             // See L81 below for an explanation.
```

Comment:
L81 is actually above. That's probably not a good way to create permalinks to code.

Reply:
Done